src/components/VolumeIndicator/VolumeIndicator.vue

```diff
@@ -102,45 +105,56 @@ export default {
 				return 0
 			}
 
-			return this.size * (1 - this.currentVolume / this.volumeThreshold)
+			return this.size * this.computeVolumeLevel(20)
 		},
+	},
 
-		overlayMutedColor() {
-			return !this.audioEnabled ? this.overlayColor : undefined
+	methods: {
+		/**
+		 * Compute volume proportion based on currentVolume [-100, 0]
+		 * and volumeThreshold. Returns number between [0, 1]
+		 *
+		 * @param {number} overload to visually increase volume level
+		 */
+		computeVolumeLevel(overload = 0) {
```

Comment:
@ShGKme please check it:
added method to compute proportion, so we can adjust range with `overload` param. Do you see any opportunities to optimize this code?

Reply:
The formula seems to be correct 👍

I added some comments in review.